Multisig support (#98)
vacuumlabs/cardano-hw-cli: docs/token-minting.md

# Token minting
Cardano offers a way to create native tokens on the blockchain. We recommend reading the guide [Discover native tokens](https://developers.cardano.org/docs/native-tokens/) before continuing.

## Creating a mint policy

> To mint some tokens you first need to define a monetary policy script as it is called in the Cardano guide, or put simply a script which says who is eligible to mint and burn tokens.

Reviewer: ```suggestion
Token minting and burning is controlled by a _monetary policy script_, i.e. a script which says who is eligible to mint and burn tokens.
```

In this guide we will create a policy which will require two signatures, one will be a public key derived on a hardware wallet, and one will be a public key created via `cardano-cli`.

### Public keys
To generate the public key on a hardware wallet a `1855H` path should be used as described in [CIP-1855](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1855/CIP-1855.md):
```sh
cardano-hw-cli address key-gen \
--path 1855H/1815H/0H \
--verification-key-file mint-hw.vkey \
--hw-signing-file mint-hw.hwsfile
```

The second key for the purpose of this demo will be a `cardano-cli` generated one.
```sh
cardano-cli address key-gen \
--normal-key \
--verification-key-file mint-cli.vkey \
--signing-key-file mint-cli.skey
```

> Note: this is just an example, you can create scripts even with two keys generated by two different hardware wallets.

Reviewer: I'd remove this.

### Policy native script
With our keys generated we can create the policy native script:
```sh
cat >policy.script <<EOF
{
  "type": "all",
  "scripts": [
    {
      "type": "sig",
      "keyHash": "$(cardano-cli address key-hash --payment-verification-key-file mint-hw.vkey)"
    },
    {
      "type": "sig",
      "keyHash": "$(cardano-cli address key-hash --payment-verification-key-file mint-cli.vkey)"
    }
  ]
}
EOF
```

### Policy id

> In the transaction we are about to create we will also be using the hash of the native script, also called the policy id. To create it we can use this command:

Reviewer: ```suggestion
In the transaction we are about to create we will also be using the hash of the native script, also called the policy id. To compute it we can use this command:
```

Reviewer: Is there a way to compute the hash that does not use hw-cli?

```sh
cardano-hw-cli transaction policyid \
--script-file policy.script \
--hw-signing-file mint-ledger.hwsfile >policyId
```

> The `--hw-signing-file` argument is optional, but enables the hardware wallet to show you the path of the public key instead of it's hash where available.

Reviewer: ```suggestion
The `--hw-signing-file` argument is optional, but enables the hardware wallet to show you the path of the public key instead of its hash where available.
```

Reviewer: "where available" is not clear
